package.json

```diff
@@ -10,7 +10,7 @@
     "compile-sass": "./bin/build-css.sh -q --quiet-deps --no-source-map",
     "watch-ts": "tsc -w",
     "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
-    "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
+    "watch-node": "DEBUG=gov-starter-server* nodemon -r dotenv/config --watch dist/ dist/server.js | bunyan -o short",
```

Comment:
I didn't add dotenv/config preload to `watch-node-feature` as all the env vars should be present in features.env. 

This should help if a dev adds new integration tests that inadvertently rely on local config. Previously this would only be detected when pushed to CI